docker-compose.yml

```
@@ -11,23 +11,12 @@ networks:
     driver: bridge
 
 services:
-  #php56:
-  #  build:
-  #    context: .docker/php56
-  #  volumes:
-  #    - ./volumes/site/www:/var/www/html
-  #    - ./.docker/php56/config/apache.conf:/etc/apache2/sites-available/000-default.conf
-  #    - ./.docker/php56/config/php.ini:/usr/local/etc/php/conf.d/custom.ini
-  #  env_file:
-  #    - .php5.env
-  #  ports:
-  #    - 8080:80
-  #  networks:
-  #    - reverse-proxy
-  #    - mysql
   wordpress:
     build:
       context: .docker/wordpress
+      args:
+        VERSION_WORDPRESS: ${VERSION_WORDPRESS:-6.4}
+        VERSION_PHP: ${VERSION_PHP:-8.3}
     restart: unless-stopped
     volumes:
       - ./volumes/wordpress:/var/www/html
```

```suggestion
      - ./volumes/wordpress:/var/www/html
    environments:
      XDEBUG_MODE: ${XDEBUG_MODE:-off}
```

And also remove the `env_file` property. The default env_file is called `.env`. If a developer wish to use xdebug as debug mode, will need to create a `.env` file (that need to stay at `.gitignore`) and override the default value of `XDEBUG_MODE`

Maybe this tip could stay in a contributing file.

---

Great suggestion but I think instead of putting this suggestion on contributing file I'll put it on dev-doc or something like this.